src/ceramic.rs

```diff
@@ -73,6 +79,16 @@ pub async fn run(op: Operation, _stdin: impl AsyncRead, stdout: impl AsyncWrite)
             };
             stdout.write_all(stream_id.to_string().as_bytes()).await?;
         }
+        Operation::StreamCreate(args) => {
+            let (stream_id, genesis_cid, genesis_block) =
+                create_stream(args.r#type, args.controller, args.deterministic).unwrap();
+            stdout
+                .write_all(
+                    format!("{}, {}, {:?}", stream_id, genesis_cid, genesis_block).as_bytes(),
```

Comment:
Is it possible to output these in a parseable format? Generally the output of the generate/create commands produces something a machine can consume. 

Would a car file make sense here? Or maybe just a multibyte encoding of the commit?

Reply:
I like the idea of emitting a CAR file. Perhaps the stream tip CAR file that's sent to the CAS. It just has a few extra fields. That's what we'd be most likely to use elsewhere.

A CAR file (or even multibase-encoded bytes for that matter) for just the commit seems less useful.

Reply:
We could also allow printing multibase-encoded CAR file bytes :)

Reply:
It actually might be useful to generate CAR bytes for a stream that the genesis commit and tip commit, maybe even all its commits 🤔 Let me know if you have a preference.

The updated code supports the stream tip CAR format that goes in anchor requests but I could separate that for use in CAS requests, and generate a different CAR for created streams.